src/main/webapp/app/exam/manage/exams/exam-detail.component.ts

```diff
@@ -83,6 +87,9 @@ export class ExamDetailComponent implements OnInit, OnDestroy {
                     this.canHaveBonus = gradingSystemResponse.body.gradeType === GradeType.GRADE;
                 }
             });
+
+            this.setExistingSummaryEntries();
+            this.fetchAndSetExamDeletionSummary();
```

Comment:
The same issues as in https://github.com/ls1intum/Artemis/pull/9186

It is "only" the instructor side, however, we still load this data from the server whenever we load the component, even though we will only need the information in the rare case that an exam will be deleted.

I think that we should only load the information when we actually open the deletion modal.

Reply:
Changed by passing an observable to the directive that fetches the summary.